src/app/shared/filter-bar/filter-bar.component.ts

```diff
@@ -24,7 +25,10 @@ export class FilterBarComponent implements OnInit, AfterViewInit, OnDestroy {
   repoChangeSubscription: Subscription;
 
   /** Selected dropdown filter value */
-  dropdownFilter: DropdownFilter = DEFAULT_DROPDOWN_FILTER;
+  dropdownFilter: DropdownFilter = FiltersStore.getInitialDropdownFilter();
+
+  /** The initial search filter value, not updated when search filter is changed */
+  initialSearchFilter: string = FiltersStore.getInitialSearchFilter();
 
```

Comment:
It is named initial because this search filter variable is not updated at all when the user types. Instead this variable is simply used to initialize what is seen by the user at the search bar when they first open a new repo.